src/Driver.ml

```diff
@@ -99,17 +99,21 @@ let mk_tmpdir_if () =
 
 (** Check whether a command exited successfully *)
 let success cmd args =
-  match Process.run cmd args with
-  | { Process.Output.exit_status = Process.Exit.Exit 0; _ } -> true
-  | _ -> false
+  try
```

Comment:
can you document here what kind of errors might be thrown?

Reply:
that is, beyond Process_erro

Reply:
`UNIX.Unix_error(20, "create_process", "which")` this is what I have if I run KReMLin outside of a Cygwin environment.

Reply:
can we catch Process_error and Unix_error (ENOENT then?
```
# ((Obj.magic 20): Unix.error);;
- : Unix.error = Unix.ENOENT
```